js/mobilemenu.js

```diff
@@ -0,0 +1,167 @@
+  },
+  {
+    name: 'Multi-Post Stories',
+    technology: '<li>CANOPY</li><li>BACK END DEV</li><li>2015</li>',
+    image: 'url(images/Snapshoot-Portfolio2.png)',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+  {
+    name: 'Tonic',
+    technology: '<li>CANOPY</li><li>BACK END DEV</li><li>2015</li>',
+    image: 'url(images/Snapshoot-Portfolio3.png)',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+  {
+    name: 'Multi-Post Stories',
+    technology: '<li>CANOPY</li><li>BACK END DEV</li><li>2015</li>',
+    image: 'url(images/Snapshoot-Portfolio4.png)',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+];
+
+function myInformation(i) {
+  document.querySelector('.name').innerText = list[i].name;
+  document.querySelector('.description').innerText = list[i].description;
+  document.querySelector('.technology').innerHTML = list[i].technology;
+  document.querySelector('#image').style.backgroundImage = list[i].image;
+}
+
+for (let i = 0; i < list.length; i += 1) {
+  openBtn[i].addEventListener('click', () => {
+    mypopUp.style.display = 'block';
+    myInformation(i);
+    document.body.style.overflow = 'hidden';
+  });
+}
+
+closeBtn.addEventListener('click', () => {
+  mypopUp.style.display = 'none';
+  document.body.style.overflow = 'visible';
+});
+
+window.onclick = function(event) {
+  if (event.target == mypopUp) {
+    mypopUp.style.display = 'none';
+    document.body.style.overflow = 'visible';
+  }
+};
+
+//validation
+submitBtn.disabled = true;
+
+const fullnameChecker = () => {
+  icon[0].style.display = 'inline-block';
+
+  if (fullname.value === '') {
+    icon[0].style.display = 'none';
+    fullname.style.borderColor = '#ffffff';
+    errorMessage[0].style.display = 'none';
+    submitBtn.disabled = true;
```

Comment:
This code was repeated down below. We should make our code [dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) as much as we can